go/libraries/doltcore/sqle/enginetest/dolt_engine_test.go

```diff
@@ -102,6 +103,14 @@ func TestSchemaOverrides(t *testing.T) {
 	RunSchemaOverridesTest(t, harness)
 }
 
+// Provide additional test coverage for toast types by running Schema Override tests
+// using toast types instead of address types.
+func TestSchemaOverridesWithToastTypes(t *testing.T) {
```

Comment:
It's nice to have this test, but it really only exercises the "inline everything" path, right?

Reply:
It would be good to have a full end-to-end test that requires externing at least one text value inserted, and using that in various SQL statements (update, delete, insert, index lookups, etc).